allennlp/nn/util.py

```diff
@@ -983,6 +983,8 @@ def flatten_and_batch_shift_indices(indices: torch.Tensor,
     offset_indices : ``torch.LongTensor``
     """
     # Shape: (batch_size)
+    if torch.max(indices) >= sequence_length or torch.min(indices) < 0:
+        raise ConfigurationError("Any elem in indices should be in a range of [0, " + str(sequence_length-1) + "].")
```

Comment:
You can write this using python 3.6 f-string formatting like this:
```
ConfigurationError(f"All elements in indices should be in range (0, {sequence_length - 1})")
```

Reply:
OK! Have changed to this format.